clustermesh-apiserver/main.go

```diff
@@ -248,6 +249,11 @@ func runApiserver() error {
 	flags.Bool(option.K8sEnableEndpointSlice, defaults.K8sEnableEndpointSlice, "Enable support of Kubernetes EndpointSlice")
 	option.BindEnv(vp, option.K8sEnableEndpointSlice)
 
+	// The default values is set to true to match the existing behavior in case
+	// the flag is not configured (for instance by the legacy cilium CLI).
+	flags.BoolVar(&cfg.enableExternalWorkloads, option.EnableExternalWorkloads, true, "Enable support for external workloads")
```

Comment:
could this default value go in in defaults.EnableExternalWorkloads?

Reply:
I personally find flags a bit easier to read when the value is inline rather than defined as a constant, since it does not require to navigate through the code to find it (assuming that it is not reused in multiple places, otherwise having a constant ensures consistency). 

I don't have any strong preference though, and the other flags defined don't seem to be consistent in that regard.

Reply:
seems like the other flags using `defaults` are also used elsewhere, so seems like it makes sense to keep it as is :+1: